Source/ProjectFirma.Web/Views/DNRUplandRegion/EditContactViewData.cs

```diff
@@ -1,13 +1,27 @@
 ﻿
+using DocumentFormat.OpenXml.Office2013.Word;
+using ProjectFirma.Web.Common;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Web.Mvc;
+using LtInfo.Common.Mvc;
+using ProjectFirma.Web.Models;
+using Person = ProjectFirma.Web.Models.Person;
+
 namespace ProjectFirma.Web.Views.DNRUplandRegion
 {
     public class EditContactViewData : FirmaUserControlViewData
     {
         public Models.DNRUplandRegion Region { get; }
+        public IEnumerable<SelectListItem> activePersonsList { get; }

```

Comment:
this isn't really all "active" people. Its only WADNR people. maybe rename to "wadnrPeopleList"